### PR TITLE
Objects Automated Test - LPS-141598 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141598

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -423,6 +423,17 @@ definition {
...
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "${panel}");
+
+		AssertElementPresent(
+			key_category = "${category}",
+			key_portlet = "${portlet}",
+			locator1 = "ApplicationsMenu#PORTLET");
+	}
+
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:436]: Hi @andreefarias 

You can use this macro: `assertObjectIsDisplayedOnControlPanel` that is in `objectPortlet.macro`

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 2]
```diff
@@ -23,12 +23,49 @@ definition {
...
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		ObjectAdmin.assertObjectInApplicationsMenu(
+			category = "Object",
+			panel = "Control Panel",
+			portlet = "Objects");
+
+		ObjectAdmin.openObjectAdmin();
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:68]: Hi @andreefarias 

You need to assert that your access to Object Admin is possible. You can validate this by checking if you have any objects created inside (by default it already has: `User` and `Commerce order`)

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias 

Please, check my comments and let me know if you have any questions.

Thanks

**Comment by andreefarias** [comment]
ci:test:sf

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**